packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -233,10 +234,8 @@ class OverflowMenu extends Component {
   static contextType = PrefixContext;
 
   static defaultProps = {
-    ariaLabel: FeatureFlags.enabled('enable-v11-release')
-      ? null
-      : 'Open and close list of options',
-    iconDescription: 'Open and close list of options',
+    ariaLabel: FeatureFlags.enabled('enable-v11-release') ? null : 'Options',
```

Comment:
This feature flag is now permanently enabled, the toggle can be removed.

Reply:
Done!